### PR TITLE
Initial template built with Copier, and manually adjusted.

### PR DESCRIPTION
**Pull request by fccoelho**
This is just the skeleton of the project and requires careful review to remove unwanted features

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+github:
```
Comment (luabida) [inline .github/FUNDING.yml:1]: This seems to be just a template file, shouldn't it be fulfilled with Mosqlimate info?

**Review by luabida — APPROVED** [review]
LGTM

File: .github/FUNDING.yml (hunk 1 above)
Comment (fccoelho) [reply .github/FUNDING.yml:1]: We can just remove this

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.10
```
Comment (esloch) [inline .github/workflows/ci.yml:41]: maybe you need to pass the python version as a string:
```
python-version: '3.10'
```

**Comment by fccoelho** [comment]
@esloch, CI is passing now

**Review by esloch — APPROVED** [review]

**Pull request merged**